features/com.google.licenses.apache_v2/feature.properties

```diff
@@ -1,6 +1,6 @@
 copyright.url = http://www.google.com/intl/en/about.html
 copyright.text =\
-(c) Copyright Google Inc. 2015. All rights reserved.\n
+(c) Copyright Google Inc. 2016, 2017. All rights reserved.\n
```

Comment:
This form is actually the EPL-style that we replaced.  I'll update it to the ASL style as [described in the license appendix](http://www.apache.org/licenses/LICENSE-2.0#apply) as we've been doing with the code.  OK?